cli

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env bash
 
-ROOT_DIR=$(dirname "$(perl -e 'use Cwd "abs_path"; print abs_path(shift)' "$0")")
+realpath() {
```

Comment:
I have added some tests onto the `main` branch which will run this command on both Ubuntu and Mac OS to make sure it works correctly in both environments, if you want to try rebasing on top of that, it might be an easy way to test the behaviour on both platforms 😉.